### ISSUE TITLE
Differentiate whether I am writing in markdown or normal in Thunderbird 60.4

### ISSUE THREAD

**Issue by nickylimjj** [adam-p/markdown-here#548]
There are no visible indicators telling the user when I am typing in Markdown or in normal mode and i often keep toggling to remind myself which mode I am in. Consider:

1.Supporting different font types for visual indicator

2.Changing markdown-here logo in toolbar when composing email

**Comment by daniel-barrows** [comment]
I agree that Markdown editing should be a *state* or *mode* which is visually indicated to the user. As it is, it seems like Markdown Here doesn't store a state. Instead, it just detects if the email looks like Markdown and performs conversions back and forth based on this detection.

I'd suggest comparing with [ThunderHTMLEdit](https://addons.thunderbird.net/en-US/thunderbird/addon/thunderhtmledit/), which adds an HTML editor. It uses tabs to switch between HTML source and visual editors, like this:

![ThunderbirdHTMLEdit screenshot](https://addons.thunderbird.net/user-media/previews/full/205/205527.png?modified=1544989820)

I'd probably prefer a dropdown menu in the toolbar instead tabs, but either one provides a visual indicator of state.
